master/buildbot/worker/base.py

```diff
@@ -17,7 +17,6 @@
 from email.message import Message
 from email.utils import formatdate
 
-from future.utils import itervalues
```

Comment:
ahah I did not see the build failure. So isort travis definitly work and my editor is misconfigured :-p